src/web/lib/decidePalette.ts

```diff
@@ -19,7 +19,7 @@ import {
 import { pillarPalette } from '@root/src/lib/pillars';
 
 const WHITE = neutral[100];
-const BLACK = text.primary;
+const BLACK = neutral[0];
```

Comment:
```suggestion
const BLACK = neutral[7];
```
My bad here - neutral.0 is #000. We wanted neutral.7 #121212